src/mgmt/srv_system.c

```diff
@@ -393,6 +397,38 @@ mgmt_svc_step_up_cb(struct ds_rsvc *rsvc)
 	return rc;
 }
 
+int
+ds_mgmt_get_ranks(struct rdb_tx *tx, struct mgmt_svc *svc,
+		  d_rank_list_t *srv_list, unsigned int flags)
+{
+	struct enum_server_arg	arg = { 0 };
```

Comment:
is the assignment = { 0 } here needed since enum_server_arg_init() does a memset on the struct contents?

Reply:
ah, ok